Source/Core/Request.js

```diff
@@ -0,0 +1,64 @@
+/*global define*/
+define([
+        'Core/defaultValue'
```

Comment:
I think this should be `./defaultValue`.

The tests pass, but I get this when I run Viewer:

![image](https://cloud.githubusercontent.com/assets/782098/12332534/ff17d136-babc-11e5-8723-053993c2e797.png)